app/src/main/java/edu/nextstep/camp/calculator/Expression.kt

```diff
@@ -0,0 +1,48 @@
+package edu.nextstep.camp.calculator
+
+/**
+ * 계산식 관리 도구
+ * Created by jeongjinhong on 2022. 07. 18..
+ */
+class Expression {
+    var expression = ""
+        private set
+
+    fun initializeValue(value: Double) {
+        expression = value.toString()
+    }
+
+    fun addOperator(operator: String): String {
+        if (expression.isNotEmpty() && expression.last().isDigit()) {
+            expression += " $operator"
+        }
+        return expression
+    }
+
+    fun addOperand(value: Int): String {
+        if (expression.isNotEmpty() && !expression.last().isDigit()) {
+            expression += " "
+        }
+        expression += value.toString()
+        return expression
+    }
+
+    fun delete(): String {
+        if (expression.isEmpty()) return expression
+        // 앞의문자가 공백이거나 소수점이면 같이 지워버리기
+        val deleteSize = if (prevCharIsBlankOrPoint()) INCLUDING_PREVIOUS else ONLY_CURRENT
+        expression = expression.substring(0, expression.length - deleteSize)
+        return expression
+    }
+
+    fun isCompletedExpression() = expression.last().isDigit()
+
+    private fun prevCharIsBlankOrPoint() =
```

Comment:
```suggestion
    private fun prevCharIsBlankOrPoint() =
```
어떤 일을 하는 함수인지 잘 이해하기가 어렵네요.
is가 먼저 오게 해서 동사구로 변경해보면 어떨까요?